app/css/bootstrap-overrides.scss

```diff
@@ -82,3 +82,28 @@ $input-icon-height: 20px;
   padding: 11px 36px;
 }
 
+
+.arbiterLabel {
```

Comment:
bootstrap-overrides.scss is more for css properties that come from bootstrap but we override. If this if for only one component, you should put it in another file and import it in the component